waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1145,19 +1146,39 @@ proc mountRlnRelayDynamic*(node: WakuNode,
   node.wakuRlnRelay = rlnPeer
   return ok(true)
 
-proc readPersistentRlnCredentials*(path: string) : RlnMembershipCredentials {.raises: [Defect, OSError, IOError, Exception].} =
-  info "Rln credentials exist in file"
+proc writeRlnCredentials*(path: string, credentials: RlnMembershipCredentials, password: string) : RlnRelayResult[void] =
+  info "Storing RLN credentials"
+  var jsonString: string
+  jsonString.toUgly(%credentials)
+  let keyfile = createKeyFileJson(toBytes(jsonString), password)
+  if keyfile.isErr():
+    return err("Error while creating keyfile for RLN credentials")
+  if saveKeyFile(path, keyfile.get()).isErr():
+    return err("Error while saving keyfile for RLN credentials")
+  return ok()
+
+proc readRlnCredentials*(path: string, password: string) : RlnRelayResult[Option[RlnMembershipCredentials]] {.raises: [Defect, IOError, Exception].} =
```

Comment:
In this proc, we do not expose the `skip` feature, where the user can select which decrypted credential to use. 
Another useful utility is to return the number of available credentials under the supplied password. 
If it is easy to integrate them into this PR, I'd suggest adding them here, otherwise, in a follow-up PR.